### PR TITLE
feat: personalized digest launch adjustments

### PR DESCRIPTION
**Pull request by capJavert**
- [x] enable worker for new users to be enrolled
- [x] adjust import script to apply to all users
- [x] adjust script for variations

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]

File: bin/assignUsersToPersonalizedDigestVariation.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  });
+
+  console.log('running db query');
+
+  const con = await createOrGetConnection();
+
+  await con.transaction(async (manager) => {
+    await manager.query(
+      `
+        UPDATE user_personalized_digest
```
Comment (idoshamun) [inline bin/assignUsersToPersonalizedDigestVariation.ts:54]: This will be one big query. I suggest splitting it into batches of 1k users roughly

Comment (capJavert) [reply bin/assignUsersToPersonalizedDigestVariation.ts:54]: Had the batch logic, I thought I'll just chunk the CSV?

**Review by idoshamun — COMMENTED** [review]
Regarding the worker, wouldn't it be better if we just create a digest entity together with the user instead of offloading it? I don't see a downside to doing it in one transaction. Doing it in the background may introduce potential race conditions.
Sorry for raising it so late in the process 🙏

File: bin/assignUsersToPersonalizedDigestVariation.ts (hunk 1 above)
Comment (idoshamun) [reply bin/assignUsersToPersonalizedDigestVariation.ts:54]: Sure that works!

Comment (capJavert) [reply bin/assignUsersToPersonalizedDigestVariation.ts]: Started to split up CSV.. figured its easier to just batch so.. added batching 0f607d889a4f9e8bf51bbd7a0ff7dac21eb31aaf

Comment (capJavert) [reply bin/assignUsersToPersonalizedDigestVariation.ts]: I tried 10k with initial import and it went instantly so I think it is ok default.

**Review by idoshamun — APPROVED** [review]

**Pull request merged**